src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
Here we can cheat a little and improve the code by declaring an array with all the column names and looping through that array just like we currently do with the `people` array.